addons/hr_homeworking/static/src/calendar/common/popover/calendar_common_popover.js

```diff
@@ -0,0 +1,34 @@
+/** @odoo-module **/
+
+import { patch } from "@web/core/utils/patch";
+import { AttendeeCalendarCommonPopover } from "@calendar/views/attendee_calendar/common/attendee_calendar_common_popover";
+import { Field } from "@web/views/fields/field"
+patch(AttendeeCalendarCommonPopover.prototype, "hr_homeworking_calendar_common_popover",{
+
+    setup() {
+        this.fieldNames = ["work_location_id", "work_location_name", "work_location_type", "employee_id", "weekday", "weekly", "date"];
+        this._super(...arguments)
+    },
+    isWorkLocationEvent(){
+        return 'resmodel' in this.props.record.rawRecord && this.props.record.rawRecord['resmodel'] === 'hr.homeworking';
```

Comment:
this should be enough
```suggestion
        return this.props.record.rawRecord['resmodel'] === 'hr.homeworking';
```